### PR TITLE
Frontend/refactor/reference tag colors

### LINKED ISSUE #1524: Reference Tag Colors [UX]
On the users' references page, the tags should be a neutral color for optimal information hierarchy. The colored tag creates an unnecessary pop-out effect even though the tags naturally pop out. 

Before
-------
Screen Shot 2021-06-27 at 9 34 08 PM

After
-----
Screen Shot 2021-06-27 at 9 34 13 PM

_Corner Radius: 4px
Fill: Light Gray (E9E9E9)
Text: Ubuntu / Bold / 12
Text Color: Almost Black (27282C)_

### PR DESCRIPTION
**Pull request by AlexVCS**
Changed the color of reference tag buttons and made the pill buttons square instead of rounded.
closes #1524

Frontend checklist

    [✅ ] Formatted my code with yarn format && yarn lint --fix
    [✅ ] There are no warnings from yarn lint
    [✅ ] There are no console warnings when running the app
    [✅ ] Clicked around my changes running locally and it works
    [✅ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: node_modules/.yarn-integrity [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (lucaslcode) [inline node_modules/.yarn-integrity]: Please remove this file too

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Looks good except for the rogue file!

File: node_modules/.yarn-integrity (hunk 1 above)
Comment (AlexVCS) [reply node_modules/.yarn-integrity]: Done :)

**Review by lucaslcode — APPROVED** [review]
Thanks!

**Pull request merged**